static/app/components/events/meta/annotatedText/utils.tsx

```diff
@@ -61,27 +60,20 @@ export function getTooltipText({
     return (
       <Wrapper>
         {tct(
-          '[method] because of the PII rule [break][rule-description] in the settings of the organization [break][slug]',
+          '[method] because of the PII rule [rule-link] in the settings of the organization [organization-slug]',
```

Comment:
A nit here, there's a bit of inconsistency with the key names. `rule-link` describes what you're render vs `organization-slug` the content of what you're rendering. The latter is better because the translator shouldn't that it's a link, but rather the link text (in this case, the rule description).

Reply:
I understand but it's difficult to think in this way because its a link 😄